src/routes/+page.svelte

```diff
@@ -138,6 +139,46 @@
 		];
 	}
 
+	// Process cards
+	let processCards: HTMLElement;
+	let leftButton: HTMLButtonElement;
+	let rightButton: HTMLButtonElement;
+
+	function scrollToProcessCard(button: "left" | "right") {
+		const processCardwidth = processCards.clientWidth;
+		const index =
+			button === "left"
+				? Math.floor(processCards.scrollLeft / processCardwidth) - 1
+				: Math.floor(processCards.scrollLeft / processCardwidth) + 1;
```

Comment:
For these two, are you sure you want to use `floor` over `round` or `ceil`? Real question, just wondering (not familiar enough with the algorithm)